helm/dex-app/templates/_helpers.tpl

```diff
@@ -69,3 +69,11 @@ Selector dex-k8s-authenticator giantswarm labels
 app.kubernetes.io/name: dex-k8s-authenticator-giantswarm
 app.kubernetes.io/component: dex-k8s-authenticator-giantswarm
 {{- end -}}
+
+{{- define "giantswarm-connector" -}}
+{{- if .Values.oidc.giantswarm.connectorConfig.clientID -}}
```

Comment:
Why wouldn't that be defined?

Reply:
because my intention here is to open to all type of clusters. So I have defaulted GS connector

Reply:
default values.yaml were already 
```
  giantswarm:
    connectorConfig:
      clientID: ""
      clientSecret: ""
      team: ""
    write_all_group: ""```

Reply:
TBH it isn't great to have `giantswarm` in values.yaml

Reply:
but this is the same chart we use for MC, so we don't need to maintain two charts. I don't want to touch any values so I ensure it does not impact MC deployments

Reply:
If Pawel's concern is mainly usability, I recommend to add some commenting that explains why this giantswarm entry is here, and what it's for.

Reply:
ok I add it now. I get the idea but I don't want to touch now the default values format, it is risky.

-